Training 453/app/css/normalize.css

```diff
@@ -0,0 +1,527 @@
+/*! normalize.css v1.1.2 | MIT License | git.io/normalize */
+
+/* ==========================================================================
```

Comment:
please be consistent with the comments format and if there are multiple comments one after another. Consider merge them into a single comment